packages/features/bookings/lib/handleNewBooking.ts

```diff
@@ -1245,6 +1245,13 @@ async function handler(
           luckyUserPool: luckyUserPool.map((user) => user.id),
         })
       );
+
+      if (reqBody.teamMemberEmail) {
+        const teamMember = eventTypeWithUsers.users.find((user) => user.email === reqBody.teamMemberEmail);
+        if (teamMember) {
+          luckyUsers.push(teamMember);
+        }
+      }
```

Comment:
If the `teamMemberEmail` is passed then let's skip the logic of finding the lucky user.